File: Serenity_Ilk_Hafta_odevi/MultiTenancy/MultiTenancy/MultiTenancy/MultiTenancy.Web/Modules/Administration/Tenant/TenantColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace MultiTenancy.Administration.Columns;

[ColumnsScript("Administration.Tenant")]
[BasedOnRow(typeof(TenantRow), CheckNames = true)]
public class TenantColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int TenantId { get; set; }
    [EditLink]
    public string TenantName { get; set; }
}

File: Serenity_Ilk_Hafta_odevi/MultiTenancy/MultiTenancy/MultiTenancy/MultiTenancy.Web/Modules/ServerTypes/Administration/TenantColumns.ts
import { ColumnsBase, fieldsProxy } from '@serenity-is/corelib';
import { Column } from '@serenity-is/sleekgrid';
import { TenantRow } from './TenantRow';

export interface TenantColumns {
    TenantId: Column<TenantRow>;
    TenantName: Column<TenantRow>;
}

export class TenantColumns extends ColumnsBase<TenantRow> {
    static readonly columnsKey = 'Administration.Tenant';
    static readonly Fields = fieldsProxy<TenantColumns>();
}